src/core/main.c

```diff
@@ -3129,6 +3129,21 @@ int main(int argc, char *argv[]) {
                         log_set_target(LOG_TARGET_JOURNAL_OR_KMSG);
 
                 } else {
+                        const char *selinuxns = getenv("SELINUXNS");
```

Comment:
hmm, isn't there a better way to detect this? do we really need this env var? i.e. so far we mounted selinuxfs read-only in containers to communicate that selinux was not supposed to be touched there. Couldn't we reuse this there, i.e, if selinuxns is on, then mount it writable, otherwise mount it read-only, as before?

(Also, we nowadays prefer to use secure_getenv() everywhere, unless we know for sure getenv() is safe and the way to go. In this case it shouldn't matter of course, as PID 1 is fully priv anyway, but as a matter of coding style, we prefer to stick to the side of paranoia)